lib/docker-file.ts

```diff
@@ -10,15 +11,31 @@ async function getBaseImageName(targetFile?: string):
   }
 
   const contents = await readFile(targetFile);
+  const dockerfile = DockerfileParser.parse(contents);
+  const from = dockerfile.getFROMs().pop();
+  let baseImage;
 
-  const results: string[] = [];
-  const FROM_RE = /^FROM ([^\s]+)[\s\S]*?$/gim;
-  let match: RegExpExecArray|null;
-  while (match = FROM_RE.exec(contents)) {
-    results.push(match[1]);
+  if (from) {
```

Comment:
```suggestion
  if (!from) {
  return {};
  }
```

Reply:
It'll need to return undefined, rather than an empty object, but sure, can reorder the code